iothub/device/tests/ConnectionString/DeviceClientConnectionStringExceptionTests.cs

```diff
@@ -14,7 +14,7 @@ public class DeviceClientConnectionStringExceptionTests
         [TestMethod]
         [TestCategory("CIT")]
         [ExpectedException(typeof(ArgumentException))]
-        public void DeviceClient_ConnectionString_DefaultScope_DefaultCredentialType_MissingEndpoint_ExceptionTest()
+        public void DeviceClientConnectionStringDefaultScopeDefaultCredentialTypeMissingEndpointExceptionTest()
```

Comment:
Is this really necessary for suppressing the warnings? It's more readable the other way.